doc/classes/Line2D.xml

```diff
@@ -82,6 +85,10 @@
 		<member name="joint_mode" type="int" setter="set_joint_mode" getter="get_joint_mode" enum="Line2D.LineJointMode" default="0">
 			The style of the connections between segments of the polyline. Use [enum LineJointMode] constants.
 		</member>
+		<member name="min_curve_line_segments" type="int" setter="set_min_curve_line_segments" getter="get_min_curve_line_segments" default="1">
+			Subdivide the line into the number of at least [member min_curve_line_segments] segments. The resulting segment count varies depending on the use of a fill gradient and the distances between points.
+			[b]Note:[/b] Using higher values results in more segments (and vertices/indices) that need to be generated which can degrade performance, particularly if you are animating properties on your line.
```

Comment:
```suggestion
			The minimum number of segments that the line should be subdivided into. The resulting segment count depends on the use of a fill gradient and the distances between points.
			[b]Note:[/b] Using higher values results in more segments (and vertices/indices) that need to be generated which can hurt performance, particularly if you are animating properties on your line.
```
Alternatively, the note below may not be necessary, and it could be merged with the first line:
```suggestion
			The minimum number of segments that the line should be subdivided into. The resulting segment count depends on the use of a fill gradient and the distances between points. Higher values result in changes to the curve's width appearing smoother, at the cost of performance. See also [member width_curve].
```